File: JavaScript/1012.js
var input = require('fs').readFileSync('stdin', 'utf8');

var [A, B, C] = input.split(" ");

var areat = (A*C)/2.0;
var areac = 3.14159*Math.pow(C, 2.0);
var areap = ((parseFloat(A)+parseFloat(B))*C)/2.0;
var areaq = Math.pow(B, 2.0);
var arear = A*B;

console.log("TRIANGULO: "+(areat).toFixed(3));
console.log("CIRCULO: "+(areac).toFixed(3));
console.log("TRAPEZIO: "+(areap).toFixed(3));
console.log("QUADRADO: "+(areaq).toFixed(3));
console.log("RETANGULO: "+(arear).toFixed(3));

File: C/1012.c
#include <stdio.h>
#include <stdlib.h>

int main() {

    double A, B, C;
    scanf("%lf", &A);
    scanf("%lf", &B);
    scanf("%lf", &C);

    double areat = (A * C) / 2.0;
    double areac = (C * C) * 3.14159;
    double areatr = ((A + B) * C) / 2.0;
    double areaq = B *B;
    double arear = A * B;

    printf("TRIANGULO: %.3lf\n", areat);
    printf("CIRCULO: %.3lf\n", areac);
    printf("TRAPEZIO: %.3lf\n", areatr);
    printf("QUADRADO: %.3lf\n", areaq);
    printf("RETANGULO: %.3lf\n", arear);

    return 0;
}